docs/guides/chrome-extensions.md

```diff
@@ -42,8 +39,40 @@ Chrome Manifest V3 extensions have a background ServiceWorker of type
 
 :::
 
+```ts
+import puppeteer from 'puppeteer';
+import path from 'path';
+
+const pathToExtension = path.join(process.cwd(), 'my-extension');
+const browser = await puppeteer.launch({
+  args: [
+    `--disable-extensions-except=${pathToExtension}`,
+    `--load-extension=${pathToExtension}`,
+  ],
+});
+
+const workerTarget = await browser.waitForTarget(
+  target => target.type() === 'service_worker' && idMatches(target)
```

Comment:
I think we should probably remove the `idMatches` call here, unless we want to document how to have an extension with a consistent ID (which seems a little out of scope?).